spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/mongo/MongoReactiveDataAutoConfiguration.java

```diff
@@ -43,12 +44,13 @@
  * to the {@literal test} database.
  *
  * @author Mark Paluch
+ * @author Yulin Qin
  * @since 2.0.0
  */
 @Configuration
 @ConditionalOnClass({ MongoClient.class, ReactiveMongoTemplate.class })
 @EnableConfigurationProperties(MongoProperties.class)
-@AutoConfigureAfter(MongoReactiveAutoConfiguration.class)
+@AutoConfigureAfter({ MongoReactiveAutoConfiguration.class, MongoDataAutoConfiguration.class })
```

Comment:
Please rebase your PR rather than changing something that has already been changed.

Reply:
done